README.md

```diff
@@ -220,27 +220,12 @@ $ npm run setup
 
 _As we use Prisma for managing our database, it is up to you what relational database framework to use._
 
-[Here](https://github.com/docker-library/docs/blob/master/postgres/README.md) is a guide to setting up [Postgres](https://www.postgresql.org/) inside a Docker container _(note: this will be a separate Docker container from the one used for running the bot itself)_.
+By default, CSBot uses SQLite. All persistent data is stored in a single file.
 
-While the choice of database is up to you, the instructions for getting started in this guide assume you are using the [Postgres Docker image](https://hub.docker.com/_/postgres).
-
-After you have Postgres (or your database of choice) up and running, edit this line in your `.env` file:
-
-```
-DATABASE_URL=postgres://{pg_user}:{pg_pass}@{pg_hostname}:{pg_port}/{pg_db}
-# required for any database functionality, we will get this URL in a later section
-```
-
-- pg_user = The Username you set in your POSTGRES_USER environment variable (default postgres)
-- pg_pass = The Password you set in your POSTGRES_PASS environment variable (default postgres)
-- ph_host = The IP of the server running your Postgres instance (default localhost)
-- pg_port = The Port assigned to your Postgres instance (default 5432)
-- pg_db = The Name of the database you wish to use for the bot
-
-The first time you run this project, you should run the following command to initialize the database:
+Edit this line in your `.env` file:
 
 ```
-$ npm run db:init
+DATABASE_URL=file:/path/to/your/database.db
```

Comment:
Can we add instructions on how to set up SQLite?

Reply:
Ohh, good point! Looks like `db:init` may still be needed. Fixed.